tests/beaker/pre-suite/00_pe_install.rb

```diff
@@ -0,0 +1,15 @@
+require 'master_manipulator'
+test_name 'FM-3804 - C61700 - Install Puppet Enterprise'
+
+# Init
+step 'Install PE'
+install_pe
```

Comment:
There is no requirement that PE be used for the master. You could use the FOSS stuff instead.